.github/workflows/ci-python.yml

```
@@ -0,0 +1,20 @@
+  push:
+    paths:
+      - '**/*.py'
+      - 'ci/reusable/lint-test-python.yml'
+  pull_request:
+    paths:
+      - '**/*.py'
+      - 'ci/reusable/lint-test-python.yml'
+
+jobs:
+  lint-test:
+    uses: ./.github/workflows/../ci/reusable/lint-test-python.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/../ci/reusable/lint-test-python.yml": open /home/jailuser/git/.github/ci/reusable/lint-test-python.yml: no such file or directory

(workflow-call)